docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+### Docker compose setup file
```

Comment:
Isn't it implied exactly what this file is?

Reply:
Line #3 and line #4.

A production docker file shouldn't have code autoreloading.

Reply:
I'm not referring to the actual content of the file, just this specific comment. The name of the file implies that it's for docker compose.

Reply:
Ah, I didn't realise this wasn't replying to my review, sorry.